tasks/40_find-friend/find-friend.js

```diff
@@ -0,0 +1,16 @@
+//  It works but I need a lesson about regular expressions.
+//  It should be refactored in more professional way
+//  with proper regex pattern.
+
+
+export const findFriend = ( arr ) => {
+  const stringified = arr.join(' ');
+  const pattern = ['blue blue red', 'red blue blue'];
```

Comment:
There is a point about this task(maybe tests not cover this) that pattern can be also `blue red blue` and in this case your solution will not work

Reply:
Test doesn't cover this. I fail to pass tests, always get one result more than is expected :)

Reply:
you will need to update your solution a bit :) there is a way with loop and if statements(easier to understand) and a few ways with regexp and some string manipulations

Reply:
We just have bad test cases here :(

Reply:
Done.